### ISSUE TITLE
FAQ/Cookbook

### ISSUE THREAD

**Issue by wshayes** [Vheissu/aurelia-for-real-world-web-applications-book-feedback#39]
I love your book so much that I want it to be everything about Aurelia which might not be the best idea :)

I was wondering if you'd be interested in putting a FAQ/Cookbook section in it.

There are a lot of basic questions that come up in Gitter, e.g.
- How do I access a value/method in the parent component?
- How do I access a value/method in the child component?
- How do I share state in my application? (pro's and con's of singleton service or event aggregation)
  ...

**Comment by Vheissu** [comment]
@wshayes This is absolutely something I had in mind @wshayes because it is publish in progress, anything is on the table. I regularly see the same questions popping up on the Gitter chat that I occasionally note down, so if you have any questions you'd like to see added into the cookbook/FAQ section, definitely feel free to note them down in here.

Btw, you've been awesome at suggesting ideas and feedback, it is much appreciated.

**Comment by opcodewriter** [comment]
Yes, the book should be more than a reference to features which (unfortunately) the official documentation does not (yet) cover at all.
The book should talk about common scenarios/tasks and their solutions, and give some forward thinking based on experience. For me this is what really good content means. However I'm not saying it's something which is easy to do :)

**Comment by zantinger** [comment]
agree with @wshayes & @opcodewriter.
